ofxProjectGenerator/src/addons/ofAddon.h

```diff
@@ -45,6 +45,7 @@ class ofAddon {
     std::vector < std::string > frameworks;		// osx only
     std::vector < std::string > data;
 	std::vector < std::string > defines;
+	std::vector < std::string > preprocessorDefinitions; // vs only
```

Comment:
this one is still missing, it should go away to. there should be no changes in ofAddon.h only on visualStudioProject

Reply:
excuse the inconvenience.